src/composables/useSearchParams.ts

```diff
@@ -42,9 +43,9 @@ export function useSearchParams() {
     // generate search params based on route query
     const params: Record<string, string> = {};
     const parsedRoute = searchQuery.parse(route);
-    search.value = parsedRoute.query.query || '';
+    query.value = parsedRoute.query.query || '';
```

Comment:
replacing the `search` with the `query` was intended?

No the search box on the NavBar will use the djangoql query instead of a normal search

Reply:
Yes, as the search facets are handled independently on a loop after this.

Reply:
But now the search bar on the navbar uses the query instead of the old search, so we lost that functionality?
![image](https://github.com/user-attachments/assets/58961174-7241-49f8-a0ff-503035789943)

Reply:
Not sure what you mean, the query is being used for the query input value. Both the individual fields search and the query filter are working to me.

What is `query=testing` supposed to return?